doc/ARCHITECTURE.md

```diff
@@ -82,3 +84,15 @@ To collect the metrics the list of github webhook events subscribed with oauth a
 - workflow_job
 
 More details about when each webhook event occurs, together with a detailed list of all metrics they trigger, see [METRICS.md](./METRICS.md).
+
+### Adding a webhook to a project
+
+In order to add a webhook to collect data of a project a new webhook has to be created within the github organization.
```

Comment:
The webhook can be created on organization level or on project level the documentation should reflect that.

I was also thinking if we should rename the parent category to something like 'Github Webhook Events & Collected Metrics', what do you think?